src/OpenCensus.Exporter.Stackdriver/Implementation/StackdriverStatsExporter.cs

```diff
@@ -88,6 +88,11 @@ internal class StackdriverStatsExporter
             displayNamePrefix = GetDisplayNamePrefix(configuration.MetricNamePrefix);
         }
 
+        static StackdriverStatsExporter()
+        {
+            USER_AGENT = $"opencensus-csharp/{Assembly.GetExecutingAssembly().GetName().Version.ToString()}";
```

Comment:
Assembly version will not include build version. So it will be constant for a long time. Consider using `AssemblyInformationalVersionAttribut` or `AssemblyFileVersion`

```
typeof(StackdriverStatsExporter).GetTypeInfo().Assembly.GetCustomAttributes<AssemblyInformationalVersionAttribute>().First().InformationalVersion;
```

![image](https://user-images.githubusercontent.com/9950081/50238347-0c5caa80-0374-11e9-88ac-b9a1e2d68add.png)

Reply:
Thanks! I wasn't sure when this is assigned (assembly information version). Will update it.

Reply:
Updated PR released.